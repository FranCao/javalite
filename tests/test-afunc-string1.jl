int main()
{
    string s;
    string r;
    s = "Hello World";
    prints(s);
    r = reverse(s);
    prints(r);
    r = reverse(r);
    prints(r);
}