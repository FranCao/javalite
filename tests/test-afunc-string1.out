Hello World
dlroW olleH
Hello World